### DISCUSSION TITLE
What is the (T) stand for (Guidance Start (T) / Guidance End (T))

### DISCUSSION THREAD

**Discussion by dtlnor** [Mikubill/sd-webui-controlnet#420] [Q&A]
and it states that Guidance End = Guidance strength, what about Guidance Start

**Comment by fmnijk** [comment]
Now the answer is in [readme](https://github.com/Mikubill/sd-webui-controlnet#weight-and-guidance-strengthstartend).

Below is what it stated:
### Weight and Guidance Strength/Start/End

Weight is the weight of the controlnet "influence". It's analogous to prompt attention/emphasis. E.g. (myprompt: 1.2). Technically, it's the factor by which to multiply the ControlNet outputs before merging them with original SD Unet.

Guidance Start/End is the percentage of total steps the controlnet applies (guidance strength = guidance end). It's analogous to prompt editing/shifting. E.g. \[myprompt::0.8\] (It applies from the beginning until 80% of total steps)

**Reply by dtlnor** [reply]
> Now the answer is in [readme](https://github.com/Mikubill/sd-webui-controlnet#weight-and-guidance-strengthstartend).
> 
> Below is what it stated:
> 
> ### Weight and Guidance Strength/Start/End
> Weight is the weight of the controlnet "influence". It's analogous to prompt attention/emphasis. E.g. (myprompt: 1.2). Technically, it's the factor by which to multiply the ControlNet outputs before merging them with original SD Unet.
> 
> Guidance Start/End is the percentage of total steps the controlnet applies (guidance strength = guidance end). It's analogous to prompt editing/shifting. E.g. [myprompt::0.8] (It applies from the beginning until 80% of total steps)


 ~~so what is the (T) stand for. ~~
